model_runner/validator/_tests/test_validator.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from pydantic import ValidationError
+
+from model_runner.validator.validator_utils import ConfigModel
+
+
+@pytest.fixture(scope="session")
+def base_config(tmp_path_factory):
+    my_file = tmp_path_factory.mktemp("data") / "myfile"
+    my_file.touch()
+
+    config = {
+        "data": tmp_path_factory._basetemp,
+        "hyperparameters": {
+            "augment": [True, False],
+            "batch_size": [2, 16, 32],
+            "lr": [5e-05, 0.001],
+        },
+        "job_array": {
+            "gpu_type": "TITANRTX",
+            "logfile_name_template": "my_job",
+            "memory": 4000,
+            "ngpus": 2,
+            "njobs": 20,
+            "njobs_parallel": 4,
+            "processor_cores": 16,
+            "run_time": "3:00",
+            "scratch": 4000,
+        },
+        "runner": my_file,
+    }
+
+    return config
+
+
+def test_good_config(base_config):
+    try:
```

Comment:
Here we should not wrap the constructor in try/except so that if there is an error, it is raised.

It can also be nice to test a couple of the fields to make sure they were validated correctly

Reply:
For example, make sure the path separator is added to the end of a directory path.